sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Result.scala

```diff
@@ -74,7 +76,8 @@ sealed trait Result[+A] extends Product with Serializable {
   }
 }
 
-final case class ResultInterruption[A](continue: () => Result[A]) extends Result[A]
+final case class ResultInterruption[A](continue: () => Result[A], abort: () => Option[String])
```

Comment:
That `abort` is more like `collectAbortInfo` which is emphasized by the return type. We are not really aborting anything in that callback.

Reply:
We are actually doing both. 
- kill the machine (see Engine.scala:422) 
- and return the stack trace.

The rational is that we want to be sure the caller uses only one of the two continuations.